introduction/the_xing_origin_story.md

```diff
@@ -1,8 +1,14 @@
-# Introduction: The Xing Origin Story
```

Comment:
I think it would be helpful to incorporate the cuts in master here. I don't mind the headings though.